subprojects/creating-new-gradle-builds/samples/code/zip.gradle.kts

```
@@ -12,6 +12,6 @@ tasks.create<Zip>("zip") {
     group = "Archive"
 
     from("src")
-    setArchiveName("basic-demo-1.0.zip")
+    setArchiveFileName("basic-demo-1.0.zip")
```

I believe that this needs to be this:
```suggestion
    archiveFileName.set("basic-demo-1.0.zip")
```

If you agree to use this suggestion, make sure you add the sign-off on the commit (unfortunately, GitHub doesn't currently support adding this by default).